src/components/BigNumber/BigNumber.tsx

```diff
@@ -49,209 +49,189 @@ const BigNumber: FC<BigNumberProps> = ({
 	};
 
 	const { chartWidth, height, locale, data, ...rscChartRemain } = rscChartProps;
-	const aspectRatio = BIG_NUMBER_ASPECT_RATIO;
-	// based on Chart.tsx checks, data will always be defined and have a length greater than 0.
 	const bigNumberValue = getBigNumberValue(data, dataKey, method);
-
 	const numberLocale = getLocale(locale).number;
-
-	const formattedValue = getFormattedString(bigNumberValue, numberType, numberLocale, numberFormat);
-
+	const formattedValue = getFormattedString(bigNumberValue, numberType, numberFormat, numberLocale);
 	const lineElements = sanitizeBigNumberChildren(children);
-	const lineProps = lineElements.length > 0 ? (lineElements[0] as LineElement).props : undefined;
-
-	let cWidth: number, cHeight: number, generalJustify: 'center' | 'start', padding: number;
 
-	if (orientation == 'vertical') {
-		padding = 0;
-		cHeight = height ? height / 2.25 : chartWidth / aspectRatio;
-		cWidth = height ? cHeight * aspectRatio : chartWidth;
-		generalJustify = 'center';
-	} else {
-		if (height && height < chartWidth / (1.75 * aspectRatio)) {
-			cHeight = height;
-			cWidth = height * aspectRatio;
-		} else {
-			cWidth = chartWidth / 1.75;
-			cHeight = cWidth / aspectRatio;
-		}
-		padding = 10;
-		generalJustify = 'start';
+	let lineProps;
```

Comment:
We want to avoid `any` types as much as possible. We should be able to type this:
```suggestion
	let lineProps: LineProps | undefined;
```